cfme/tests/openstack/cloud/test_volume_backup.py

```diff
@@ -72,3 +129,20 @@ def test_create_volume_backup(volume_backup):
 def test_create_volume_incremental_backup(incremental_backup):
     assert incremental_backup.exists
     assert incremental_backup.size == VOLUME_SIZE
+
+
+def test_incr_backup_of_attached_volume_crud(provider, incremental_backup_of_attached_volume):
+    assert incremental_backup_of_attached_volume.exists
```

Comment:
I think the code that's in the fixture providing the backup and attaching the volume belongs here in the test, not in the fixture.

The test is CRUD, which includes creating the backup - so it shouldn't be in the setup phase.

Attaching the volume to the disk, that can stay in the fixture, because that is setup for being able to take backups of the volume.